src/Domain/Model/MembershipApplication.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Domain\Model;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class MembershipApplication {
+
+	/**
+	 * @var int
+	 */
+	private $id;
```

Comment:
Needs to be nullable :( Unless we do something different than having the repository assign the ID on write